lib/fittings/json_error_handler.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+var debug = require('debug')('swagger:json_error_handler');
+var util = require('util');
+
+module.exports = function create(fittingDef, bagpipes) {
```

Comment:
'bagpipes' is defined but never used.<br>'fittingDef' is defined but never used.